src/main/java/org/opentripplanner/middleware/controllers/api/OtpUserController.java

```diff
@@ -52,6 +52,24 @@ protected void buildEndpoint(ApiEndpoint baseEndpoint) {
         super.buildEndpoint(modifiedEndpoint);
     }
 
+    /**
+     * HTTP endpoint to get the {@link OtpUser} entity, if it exists, from an {@link Auth0UserProfile} attribute
+     * available from a {@link Request} (this is the case for '/api/secure/' endpoints).
+     */
+    private OtpUser getUserFromRequest(Request req, Response res) {
```

Comment:
As mentioned on Teams, maybe we should have a single endpoint that returns all of the various user types?

Reply:
Noted in #40.